app/src/main/res/layout/activity_home.xml

```diff
@@ -42,21 +42,19 @@
         android:inputType="text"
         android:textAlignment="center" />
 
-    <Button
-        android:id="@+id/buttonContinue"
+    <androidx.appcompat.widget.AppCompatSpinner
+        android:id="@+id/audioModeSpinner"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="@string/join_with_audio"
-        android:contentDescription="@string/main_continue"
-        android:gravity="center"
+        android:contentDescription="@string/audio_mode_spinner"
         android:layout_gravity="center"/>
 
     <Button
-        android:id="@+id/buttonContinueWithoutAudio"
+        android:id="@+id/buttonContinue"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="@string/join_without_audio"
-        android:contentDescription="@string/main_continue_without_audio"
+        android:text="@string/join_meeting"
+        android:contentDescription="@string/main_continue"
```

Comment:
Will the integration tests updated with new UI?

Reply:
No new integration tests were added for Join Without Audio when it was a separate button. The integration test will continue to work for regular Join Meeting flow.

Reply:
Will adding new integ test for this audio be a separate task?

Reply:
We haven't planned that task.